keyboards/dcpedit/midevil/readme.md

```diff
@@ -0,0 +1,27 @@
+# MidEvil
+
+![MidEvil](https://i.imgur.com/WCE6GIU.jpg)
```

Comment:
```suggestion
![MidEvil](https://i.imgur.com/WCE6GIUh.jpg)
```

Reply:
I had to revert this change because the new URL is now returning 404